h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -129,6 +132,13 @@ export default function AnnotationCard({
 
   const annoQuote = quote(annotation);
 
+  // Should the annotation body be rendered in a collapsed state, assuming it is
+  // collapsible (exceeds allotted collapsed space)?
+  const [annoBodyCollapsed, setAnnoBodyCollapsed] = useState(true);
+  // Does the annotation body take up enough vertical space that
+  // collapsing/expanding is relevant?
```

Comment:
```suggestion
  // collapsing/expanding is enabled?
```

Changed the term to make it clearer what user affordances are affected by this state.